sycl/include/sycl/ext/oneapi/device_global/properties.hpp

```diff
@@ -17,6 +17,9 @@ namespace ext {
 namespace oneapi {
 namespace experimental {
 
+// Forward declaration.
+template <typename T, typename PropertyListT> class device_global;
```

Comment:
nit:

```suggestion
template <typename T, typename PropertyListT> class device_global;
```

C++ syntax is clear about what it is :P

Reply:
Force of habit! Has been removed.